python/tox.ini

```diff
@@ -93,7 +125,7 @@ commands =
     python -m http.server {posargs}
 
 [flake8]
-ignore = E501,W503
+ignore = E501,E721,I100,I101,I202,W503
```

Comment:
The `I` ones are the suggested ones to prevent conflicting with isort and should also include `I201` but I totally agree, let's add them when once they cause the tests to fail. Updating now.

Reply:
SG, thanks!